src/hotel-search/sorting.js

```diff
@@ -12,6 +12,34 @@ module.exports = {
       }
     }
 
+    function _getDiscount(hotel) {
+      if (_hasRates(hotel) && _hasUsualPrice(hotel.rates[0]['usualPrice'])) {
+        return Math.round(hotel.rates[0]['usualPrice']['discountToUsualAmount'] * 100);
+      } else {
+        return null;
+      }
+    }
+
+    function _getSavings(hotel) {
+      var usualPrice;
```

Comment:
can declar inside if?

Reply:
It should be fine, It will be hoisted anyway.